utils/decorator.py

```diff
@@ -0,0 +1,22 @@
+import json
+import bcrypt
+import jwt
+
+from user.models import User
+
+from my_settings import SECRET_KEY, ALGORITHM
+
+def authorization (func):
+    def wrapper (self, request, *args, **kwrgs):
+        try:
+            token = request.headers['Authorization']
+            decoded_token = jwt.decode(token, SECRET_KEY, ALGORITHM)
+            request.user = User.objects.get(id=decoded_token["user_id"])
+            return func(self, request, *args, **kwargs)
+            
+        except ValueError:
+            return JsonResponse({'message':'It returned None instead.'}, status=400)
+
+        except jwt.DecodeError:
+            return JsonResponse({'message': 'Not exact token'}, status=400)
```

Comment:
```suggestion
            return JsonResponse({'message' : 'INVALID_TOKEN'}, status=?)
```
status code를 400을 사용 했는데, 400말고 현재 상황에서 어떠한 status 코드를 사용 할 수 있을까요?

Reply:
`401 Unauthorized`가 맞는거 같습니다!

토큰이 유효하지 않다는 것은 토큰에 담겨있는 유저 정보도 유효하지 않다고 생각했고, 즉 비인증 상태를 의미한다고 생각해서 401로 수정하였습니다